Modules/Microsoft365DSC/DSCResources/MSFT_TeamsTeam/MSFT_TeamsTeam.psm1

```diff
@@ -203,6 +203,10 @@ function Get-TargetResource
             Ensure                            = 'Present'
         }
 
+        if($null -ne $Owners){
+            $result.Add('Owner',$Owners[0].User.ToString())
```

Comment:
This aligns with the previous implementation. Why are we only grabbing the first user and not all owners of a team?